### ISSUE TITLE
Add Input Field inside of Card

### ISSUE THREAD

**Issue by underdoeg** [marmelab/admin-on-rest#214]
Is it possible to have something like this in the edit view?

```
<Card>
	<CardHeader title="coordinates" />
	<CardText >
		<TextInput source="x" type="number" />
		<TextInput source="y" type="number" />
	</CardText>
</Card>
```

**Comment by fzaninotto** [comment]
No, it's not, and it concerns me, too. I'll post a suggestion for a syntax change for comments.

**Comment by underdoeg** [comment]
ok. thanks. Would be very useful when you have more than just a few input fields.

**Comment by fzaninotto** [comment]
See #216, feel free to comment.

**Comment by fzaninotto** [comment]
Fixed by #221
